### PR TITLE
Bumped version to 0.7.3 and use elastic1 for cluster name

### PR DESCRIPTION
**Pull request by t0ffel**
Now building from Centos image, not debian.
Removed unneeded IS and add unzip to the image

### CODE REVIEW

File: cerebro-template.yaml [hunk 1]
```diff
@@ -16,13 +16,13 @@ objects:
         name: openshift-cerebro:latest
     source:
       git:
-        uri: https://github.com/jcantrill/openshift-cerebro
+        uri: https://github.com/t0ffel/openshift-cerebro
```
Comment (jcantrill) [inline cerebro-template.yaml]: Looks like we should make this a template param

File: conf/application.conf [hunk 2]
```diff
@@ -50,7 +50,7 @@ auth = {
 # A list of known hosts
 hosts = [
   {
-    host = "https://logging-es:9200"
+    host = "https://elastic1:9200"
```
Comment (jcantrill) [inline conf/application.conf]: Is it possible to parameterize this too?

File: cerebro-template.yaml (hunk 1 above)
Comment (t0ffel) [reply cerebro-template.yaml]: updated

File: conf/application.conf (hunk 2 above)
Comment (t0ffel) [reply conf/application.conf]: did a quick parameterization

**Review by jcantrill — APPROVED** [review]

**Pull request merged**